plugins/docker_binary_builder/app/models/binary_builder.rb

```diff
@@ -0,0 +1,136 @@
+class BinaryBuilder
+  DOCKER_BUILD_FILE = 'Dockerfile.build'.freeze
+  BUILD_SCRIPT = '/app/build.sh'.freeze
+  ARTIFACTS_FILE = 'artifacts.tar'.freeze
+  ARTIFACTS_FILE_PATH = "/app/#{ARTIFACTS_FILE}".freeze
+  DOCKER_HOST_CACHE_DIR = '/opt/samson_build_cache'.freeze
```

Comment:
We intentionally made it shared so that all Maven/SBT/etc.. based projects would benefit from each other's cache.